WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/PaymentSettingsFlowHint.swift

```diff
@@ -0,0 +1,22 @@
+
+import SwiftUI
+
+struct PaymentSettingsFlowHint: View {
+    let title: String
+    let text: String
+
+    var body: some View {
+        HStack {
+            Text(title)
+                .font(.callout)
+                .padding(.all, 12)
+                .background(Color(UIColor.systemGray6))
+                .clipShape(Circle())
+            Text(text)
+                .font(.callout)
+                .padding(.leading, 16)
+            Spacer()
+        }
+            .padding(.horizontal, 8)
+    }
+}
```

Comment:
We can add a Preview here:

```suggestion
}

struct PaymentSettingsFlowHint_Previews: PreviewProvider {
    static var previews: some View {
        PaymentSettingsFlowHint(title: "0", text: "This is some text that acts as a hint.")
    }
}
```

On a different note: Should we rename this back to `Hint` or something like `HintView` and move it to `ReusableViews`?

Reply:
> On a different note: Should we rename this back to Hint or something like HintView and move it to ReusableViews?

I've not thought deeply about it as general-use view. 

Perhaps at some point we should, but doing that for a view which is only used in payments and isn't part of any particular design system (AFAIK) would make it less obvious where it's actually used. It felt like premature generalisation, but I did considered it.

Reply:
Added previews in 16815d4